docs/source/example_use.rst

```diff
@@ -49,7 +49,7 @@ If you haven't followed the :doc:`getting started guide <getting_started>` to
 set up Orchestra yet, you should do that now. Also, make sure that
 ``'journalism_workflow'`` is in your ``INSTALLED_APPS`` setting, and that you
 have loaded the workflow into the database (``python manage.py loadworkflow
-journalism_workflow journalism_v1``).
+journalism_workflow v1``).
```

Comment:
i like calling out the difference.  one other way to do this is with `--app` and `--workflow-version` flags. the only reason to have `workflow/version` format is when we want to allow lists of those rather than individual values.  what do you think?